src/web-components/vf-score.js

```diff
@@ -0,0 +1,137 @@
+// ## Description
+// 
+// This file implements `vf-score`, the web component that acts as the 
+// container for the entire component. `vf-score`'s shadow root holds
+// the HTML element that the renderer renders too.
+// 
+// All actual drawing is called from `vf-score`. 
+
+import Vex from '../index';
+
+export class VFScore extends HTMLElement {
+  constructor() {
+    super();
+
+    // Defaults
+    this.x = 10; // The starting x position of a system within the score.
```

Comment:
Since you added the Babel class-fields plugin, and class fields are supported in all current browsers except Safari (which has them in technology preview) - you might be able to use them here. Class fields are a better place to add jsdocs, since you can use `/** */` docs on the field.